plugins/module_utils/pacemaker.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Dexter Le <dextersydney2001@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.community.general.plugins.module_utils.cmd_runner import CmdRunner, cmd_runner_fmt
+
+
+_state_map = {
+    "present": "create",
+    "absent": "remove",
+    "status": "status"
+}
+
+
+def fmt_resource_type(value):
+    cmd = []
+    for key in ['resource_standard', 'resource_provider', 'resource_name']:
+        if value.get(key) is not None:
+            cmd.append(value.get(key))
+    return cmd
+
+
+def fmt_resource_operation(value):
```

Comment:
I did try to make this more "pythonic" as well but it is less straightforward. I reckon it is probably better to read/maintain it like this. ;-)